crates/rocksdb/src/db_manager.rs

```diff
@@ -43,9 +44,11 @@ pub struct RocksDbManager {
     cache: Cache,
     // auto updates to changes in common.rocksdb_memory_limit and common.rocksdb_memtable_total_size_limit
     write_buffer_manager: WriteBufferManager,
-    dbs: DashMap<(Owner, DbName), Arc<RocksDb>>,
+    dbs: RwLock<HashMap<(Owner, DbName), Arc<RocksDb>>>,
```

Comment:
Is `parking_lot::RwLock<HashMap<>>` faster than `DashMap` or is there something you can only do with the `RwLock`?

Reply:
Just to reduce dependencies, the original direction implicitly assumed large number of database. In practice, the number of cases that needs to access this map directly is so small that it's not worth the the potential dead-locks that come with dashmap.